requirements.txt

```diff
@@ -3,7 +3,7 @@ bioblend>=1.0.0
 click!=8.0.2
 cwltool>=1.0.20191225192155
 ephemeris>=0.10.3
-galaxy-tool-util[edam]>=23.1,!=24.0.0,!=24.0.1,<24.2
+galaxy-tool-util[edam,extended-assertions]>=23.1,!=24.0.0,!=24.0.1,<24.2
```

Comment:
You probably need to specify a minimum version where `extended-assertions` was added ?